docs/services/ml/_index.md

```diff
@@ -147,6 +147,19 @@ In the absence of metadata, your `.tflite_cpu` model must satisfy the following
 These requirements are satisfied by a few publicly available model architectures including EfficientDet, MobileNet, and SSD MobileNet V1.
 You can use one of these architectures or build your own.
 
+## Use the ML model service with the Viam Python SDK
+
+To use the ML model service from the [Viam Python SDK](https://python.viam.dev/), you must install both the Python SDK itself and the `mlmodel` extra:
+
+```sh {class="command-line" data-prompt="$"}
+pip install viam-sdk
+pip install 'viam-sdk[mlmodel]'
```

Comment:
installing `viam-sdk[mlmodel]` includes the base `viam-sdk` itself, so you don't need both commands here